### PR TITLE
Adding mongocxx instructions

### PR DESCRIPTION
**Pull request by ugocupcic**
from https://github.com/TheBrewCrew/warehouse_ros_mongo/pull/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
...
+
+First get the driver:
+```
+git clone -b 26compat https://github.com/mongodb/mongo-cxx-driver.git
+```
+
+Then compile using scons:
+```
+sudo apt-get install scons
+scons --prefix=/usr/local/ --full --use-system-boost --disable-warnings-as-errors
```
Comment (davetcoleman) [inline README.md]: this should also be sudo (unless your on Docker) as already corrected http://moveit.ros.org/install/source/

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
...
+In order to build from source you'll need to install the [mongo c++ drivers](https://github.com/mongodb/mongo-cxx-driver/wiki/Download-and-Compile-the-Legacy-Driver)
+
+First get the driver:
+```
+git clone -b 26compat https://github.com/mongodb/mongo-cxx-driver.git
+```
+
+Then compile using scons:
+```
+sudo apt-get install scons
```
Comment (davetcoleman) [inline README.md:16]: specifying `cd mongo-cxx-driver` might be good. too obvious?

File: README.md [hunk 3]
```diff
@@ -1 +1,21 @@
 Code for persisting ROS message data using MongoDB.  Contains C++ and Python libraries to serialize ROS data to MongoDB, as well as some handy scripts to record data from the command line.  Based on code split out of warehouse_ros.
+
+# Building from source
+
+## ROS Jade
```
Comment (davetcoleman) [inline README.md]: or Kinetic/or newer.

**Review by davetcoleman — CHANGES_REQUESTED** [review]

**Comment by ugocupcic** [comment]
@davetcoleman I addressed all your comments. Cheers

**Comment by davetcoleman** [comment]
Thanks so much!

**Pull request merged**